runtime/jcl/common/java_lang_StackWalker.cpp

```diff
@@ -31,11 +31,17 @@
 #include "VMHelpers.hpp"
 
 extern "C" {
-#define RETAIN_CLASS_REFERENCE 1
-#define SHOW_REFLECT_FRAMES 2
-#define SHOW_HIDDEN_FRAMES 4
-#define FRAME_VALID 8
-#define FRAME_FILTER_MASK (RETAIN_CLASS_REFERENCE | SHOW_REFLECT_FRAMES | SHOW_HIDDEN_FRAMES)
+/* Internal constant flags, equivalent Java flags defined in StackWalker class. */
```

Comment:
Of the comments here and in `StackWalker.java` one should make it clear which place is definitive.
Also I suggest "constant flags" should be "flag constants".